egs/tunisian_msa/s5/local/buckwalter2utf8.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env perl
+# Input buckwalter encoded Arabic and print it out as utf-8 encoded Arabic.
+use strict;
+use warnings;
+use Carp;
+
+use Encode::Arabic::Buckwalter;         # imports just like 'use Encode' would, plus more
```

Comment:
On Ubuntu, package "libencode-arabic-perl" needs to be installed (https://packages.ubuntu.com/bionic/libencode-arabic-perl). Perhaps this requirement can be added to the README file (just like some recipes require ffmpeg to be installed).